evento/src/views/MyEvents/index.js

```diff
@@ -17,16 +17,18 @@ class MyEvents extends Component {
 	}
 	
 	fetchEvents() {
-		const userId = Cookie.get('userId');
+		let user = Cookie.get('user');
```

Comment:
Yeah no. We aren't gonna keep both the JSON presentation and the parsed value in the same variable. That is just a bad practice and confusing. Rename this to `const userCookie` or something